test/e2e/page-objects/flows/login.flow.ts

```diff
@@ -24,12 +24,12 @@ export const loginWithoutBalanceValidation = async (
  * This method unlocks the wallet and verifies that the user lands on the homepage with the expected balance. It is designed to be the initial step in setting up a test environment.
  *
  * @param driver - The webdriver instance.
- * @param ganacheServer - The ganache server instance
+ * @param localBlockchainServer - The local blockchain server instance
  * @param password - The password used to unlock the wallet.
  */
 export const loginWithBalanceValidation = async (
   driver: Driver,
-  ganacheServer?: Ganache,
+  localBlockchainServer?: Ganache,
```

Comment:
this will be changed in the main migration PR, to accept both Ganache or Anvil classes. See [here](https://github.com/MetaMask/metamask-extension/pull/27246/files#diff-af94879a75170f2f9e28710b48669b2ddfa353d7220fd725df3d44574f0eaff9R33)